configure

```diff
@@ -896,7 +895,6 @@ datadir='${datarootdir}'
 sysconfdir='${prefix}/etc'
 sharedstatedir='${prefix}/com'
 localstatedir='${prefix}/var'
-runstatedir='${localstatedir}/run'
```

Comment:
Unrelated changes (you use different version of autotools).

Reply:
I have no idea how to prevent such unrelated change. It was discussed multiple times, no solution was found yet.

Reply:
The simplest way is to remove unrelated changes manually.